[INFO] [2020-10-29 08:12:03,376] [2468:140422157502272] - task_executor.py[line:121]: run 202010290812019279649 dataio_0 202010290812019279649_dataio_0 guest 10000 task
[INFO] [2020-10-29 08:12:03,377] [2468:140422157502272] - task_executor.py[line:122]: {'DataIOParam': {'input_format': 'dense', 'delimitor': ',', 'data_type': 'float64', 'exclusive_data_type': None, 'tag_with_value': False, 'tag_value_delimitor': ':', 'missing_fill': False, 'default_value': 0, 'missing_fill_method': None, 'missing_impute': None, 'outlier_replace': False, 'outlier_replace_method': None, 'outlier_impute': None, 'outlier_replace_value': 0, 'with_label': True, 'label_name': 'y', 'label_type': 'int', 'output_format': 'dense'}, 'initiator': {'role': 'guest', 'party_id': 10000}, 'job_parameters': {'work_mode': 0, 'model_id': 'arbiter-10000#guest-10000#host-10000#model', 'model_version': '202010290812019279649'}, 'role': {'guest': [10000], 'host': [10000], 'arbiter': [10000]}, 'config': '/fate/examples/federatedml-1.x-examples/homo_logistic_regression/test_homolr_evaluate_job_conf.json', 'dsl': 'examples/federatedml-1.x-examples/homo_logistic_regression/test_homolr_evaluate_job_dsl.json', 'function': 'submit_job', 'local': {'role': 'guest', 'party_id': 10000}, 'CodePath': 'federatedml/util/data_io.py/DataIO', 'module': 'DataIO'}
[INFO] [2020-10-29 08:12:03,377] [2468:140422157502272] - task_executor.py[line:123]: {'data': {'data': ['args.train_data']}}
[DEBUG] [2020-10-29 08:12:03,378] [2468:140422157502272] - profile_util.py[line:34]: task_executor.py:187 call table, takes 0.0005702972412109375s
[INFO] [2020-10-29 08:12:03,378] [2468:140422157502272] - task_executor.py[line:202]: start save as task 202010290812019279649_dataio_0 input data table experiment breast_homo_guest
[DEBUG] [2020-10-29 08:12:03,379] [2468:140422157502272] - profile_util.py[line:34]: session.py:178 call collect, takes 0.0005366802215576172s
[DEBUG] [2020-10-29 08:12:03,401] [2468:140422157502272] - profile_util.py[line:34]: task_executor.py:212 call save_as, takes 0.021899700164794922s
[INFO] [2020-10-29 08:12:03,404] [2468:140422157502272] - task_executor.py[line:218]: save as task 202010290812019279649_dataio_0 input data table to 202010290812019279649_dataio_0_guest_10000 breast_homo_guest done
[DEBUG] [2020-10-29 08:12:03,558] [2468:140422157502272] - component_properties.py[line:67]: {'DataIOParam': {'input_format': 'dense', 'delimitor': ',', 'data_type': 'float64', 'exclusive_data_type': None, 'tag_with_value': False, 'tag_value_delimitor': ':', 'missing_fill': False, 'default_value': 0, 'missing_fill_method': None, 'missing_impute': None, 'outlier_replace': False, 'outlier_replace_method': None, 'outlier_impute': None, 'outlier_replace_value': 0, 'with_label': True, 'label_name': 'y', 'label_type': 'int', 'output_format': 'dense'}, 'initiator': {'role': 'guest', 'party_id': 10000}, 'job_parameters': {'work_mode': 0, 'model_id': 'arbiter-10000#guest-10000#host-10000#model', 'model_version': '202010290812019279649'}, 'role': {'guest': [10000], 'host': [10000], 'arbiter': [10000]}, 'config': '/fate/examples/federatedml-1.x-examples/homo_logistic_regression/test_homolr_evaluate_job_conf.json', 'dsl': 'examples/federatedml-1.x-examples/homo_logistic_regression/test_homolr_evaluate_job_dsl.json', 'function': 'submit_job', 'local': {'role': 'guest', 'party_id': 10000}, 'CodePath': 'federatedml/util/data_io.py/DataIO', 'module': 'DataIO'}
[DEBUG] [2020-10-29 08:12:03,559] [2468:140422157502272] - component_properties.py[line:74]: need_run: True, need_cv: False
[DEBUG] [2020-10-29 08:12:03,559] [2468:140422157502272] - component_properties.py[line:118]: has_train_data: False, has_eval_data: False, has_normal_data: True
[DEBUG] [2020-10-29 08:12:03,561] [2468:140422157502272] - profile_util.py[line:34]: component_properties.py:155 call count, takes 0.0019805431365966797s
[DEBUG] [2020-10-29 08:12:03,561] [2468:140422157502272] - model_base.py[line:210]: In extract_data, data input: {'args.data': <arch.api.impl.based_1x.table.DTable object at 0x7fb17aa83fd0>}
[DEBUG] [2020-10-29 08:12:03,561] [2468:140422157502272] - model_base.py[line:87]: func: <bound method DataIO.fit of <federatedml.util.data_io.DataIO object at 0x7fb17aa83668>>
[INFO] [2020-10-29 08:12:03,561] [2468:140422157502272] - data_io.py[line:141]: start to read dense data and change data to instance
[DEBUG] [2020-10-29 08:12:03,563] [2468:140422157502272] - profile_util.py[line:34]: abnormal_detection.py:23 call count, takes 0.0008912086486816406s
[DEBUG] [2020-10-29 08:12:03,563] [2468:140422157502272] - data_io.py[line:95]: header is y,x0,x1,x2,x3,x4,x5,x6,x7,x8,x9,x10,x11,x12,x13,x14,x15,x16,x17,x18,x19,x20,x21,x22,x23,x24,x25,x26,x27,x28,x29
[DEBUG] [2020-10-29 08:12:03,563] [2468:140422157502272] - data_io.py[line:96]: sid_name is id
[DEBUG] [2020-10-29 08:12:03,605] [2468:140422157502272] - profile_util.py[line:34]: data_io.py:155 call mapValues, takes 0.04093790054321289s
[DEBUG] [2020-10-29 08:12:03,620] [2468:140422157502272] - profile_util.py[line:34]: data_io.py:158 call mapValues, takes 0.015290260314941406s
[DEBUG] [2020-10-29 08:12:03,645] [2468:140422157502272] - profile_util.py[line:34]: data_io.py:241 call join, takes 0.023773670196533203s
[DEBUG] [2020-10-29 08:12:03,645] [2468:140422157502272] - model_base.py[line:99]: saved_result is : [<arch.api.impl.based_1x.table.DTable object at 0x7f4d73068550>], data_output: <arch.api.impl.based_1x.table.DTable object at 0x7f4d73068550>
[DEBUG] [2020-10-29 08:12:03,680] [2468:140422157502272] - profile_util.py[line:34]: tracking_manager.py:201 call save_as, takes 0.03482341766357422s
[DEBUG] [2020-10-29 08:12:03,682] [2468:140422157502272] - profile_util.py[line:34]: session.py:178 call collect, takes 0.001085042953491211s
[DEBUG] [2020-10-29 08:12:03,739] [2468:140422157502272] - profile_util.py[line:34]: tracking_manager.py:221 call count, takes 0.015769004821777344s
[INFO] [2020-10-29 08:12:03,747] [2468:140422157502272] - pipelined_model.py[line:72]: Save dataio_0 dataio DataIOMeta buffer
[INFO] [2020-10-29 08:12:03,748] [2468:140422157502272] - pipelined_model.py[line:72]: Save dataio_0 dataio DataIOParam buffer
[INFO] [2020-10-29 08:12:03,752] [2468:140422157502272] - pipelined_model.py[line:77]: Save dataio_0 dataio successfully
[INFO] [2020-10-29 08:12:03,774] [2468:140422157502272] - task_executor.py[line:160]: task 202010290812019279649_dataio_0 guest 10000 start time: 2020-10-29 08:12:03
[INFO] [2020-10-29 08:12:03,774] [2468:140422157502272] - task_executor.py[line:161]: task 202010290812019279649_dataio_0 guest 10000 end time: 2020-10-29 08:12:03
[INFO] [2020-10-29 08:12:03,774] [2468:140422157502272] - task_executor.py[line:162]: task 202010290812019279649_dataio_0 guest 10000 takes 0.624s
[INFO] [2020-10-29 08:12:03,774] [2468:140422157502272] - task_executor.py[line:164]: finish 202010290812019279649 dataio_0 202010290812019279649_dataio_0 guest 10000 success task
